src/libslic3r/PrintConfig.cpp

```diff
@@ -2536,7 +2536,7 @@ def = this->add("filament_loading_speed", coFloats);
             (void)L("Maximum jerk of the Y axis");
             (void)L("Maximum jerk of the Z axis");
             (void)L("Maximum jerk of the E axis");
-            def->sidetext = L("mm/s");
+            def->sidetext = L("mm/s³");
```

Comment:
it should me mm/s

Reply:
Doesn't the term "jerk" in this context refer to the [physics concept](https://en.wikipedia.org/wiki/Jerk_%28physics%29) of change in acceleration over time? If so, that's definitely mm/s³. If not, I'd be really interested to know what it represents, because that's how I've been thinking about it.

That being said, my intent here is clarity, so either way if the change is less-clear, then I'd be happy to remove it.

Reply:
Well, it's slightly different in 3d printer.

Quoted from Marlin firmware:
>  Jerk is the maximum change in velocity (in mm/sec) that can occur instantaneously. 

In Klipper, it equals to SCV(Square corner velocity).
So the unit is mm/s

Reply:
That's good to know! Fixed.